### ISSUE TITLE
Support for glob patterns

### ISSUE THREAD

**Issue by mskelton** [Roang-zero1/github-upload-release-artifacts-action#1]
First of all, thanks so much fo this great project as it has made it so easy to use GitHub actions for uploading files to GitHub releases!

One thing that would be super helpful would be to have support for file glob patterns.  The reason this is important for me is because I'm using this extension to upload VS Code extension files which are placed in the root directory of the project when built and they contain a version number in them thus making a hard coded approach difficult.  Unfortunately, the VS Code extension cli doesn't have an easy way to place the file in a subdirectory while maintaining the version number automatically.  My solution currently looks like this.

```yaml
    steps:
      - uses: actions/checkout@master
      - uses: actions/setup-node@v1
        with:
          node-version: 10
      - run: yarn
      - run: |
          mkdir dist
          yarn vsce package -o dist/one-dark-theme-${GITHUB_REF/refs\/tags\//}.vsix
      - uses: Roang-zero1/github-upload-release-artifacts-action@master
        with:
          args: dist
        env:
          GITHUB_TOKEN: ${{ secrets.github_auth_token }}
```

By adding glob patterns I could do it like this.

```yaml
    steps:
      - uses: actions/checkout@master
      - uses: actions/setup-node@v1
        with:
          node-version: 10
      - run: yarn
      - run: yarn vsce package
      - uses: Roang-zero1/github-upload-release-artifacts-action@master
        with:
          args: '*.vsix'
        env:
          GITHUB_TOKEN: ${{ secrets.github_auth_token }}
```

**Comment by Roang-zero1** [comment]
Sorry for the late reply, notifications were turned off.
This is a great suggestion, I will look into adding it.

**Comment by Roang-zero1** [comment]
Fixed in v2.1.0

**Comment by mskelton** [comment]
Thanks so much!
